client/src/main.jsx

```diff
@@ -24,4 +25,14 @@ const router = createBrowserRouter([
 const rootElement = document.getElementById("root");
 const root = createRoot(rootElement);
 
-root.render(<RouterProvider router={router} />);
+root.render(
+  <Auth0Provider
+    domain="dev-gh7vd2ytg45l280b.us.auth0.com"
```

Comment:
Move this inside env file

Reply:
build: new env file and moved all the variables into it.